osbs/cli/main.py

```diff
@@ -263,6 +263,11 @@ def cmd_cancel_build(args, osbs):
 
 
 def cmd_build(args, osbs):
+    if args.labels:
+        labels = dict(args.labels)
+    else:
+        labels = None
```

Comment:
Python doesn't have the concept of block scoping. In this case it's function scope.